src/components/PermissionsManagement/UserPermissionsPopUp.jsx

```diff
@@ -81,7 +85,13 @@ function UserPermissionsPopUp({
 
     const url = ENDPOINTS.USER_PROFILE(userId);
     const allUserInfo = await axios.get(url).then(res => res.data);
-    const newUserInfo = { ...allUserInfo, permissions: { frontPermissions: userPermissions } };
+    const newUserInfo = {
+      ...allUserInfo,
+      permissions: {
+        frontPermissions: userPermissions,
+        removedDefaultPermissions: { front: userRemovedDefaultPermissions },
```

Comment:
```suggestion
        removedDefaultPermissions: userRemovedDefaultPermissions,
```
See comment on backend